modules/docker/README.md

```diff
@@ -26,6 +30,7 @@ Defines [Docker][1] aliases and functions.
 - `dkpl` Pull an image or a repository from a registry
 - `dkph` Push an image or a repository to a registry
 - `dkps` List containers
+- `dkps!` Display the first matching docker container ID by name
```

Comment:
Adding a `!` in an alias for an optional parameter seems little non-standard. How about replacing `dkps` alias with a function that takes an optional name?

Reply:
You're right. These new bang aliases are not mature yet. I've removed them from this PR. I'll submit something more robust and well named later on ;)